terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
+  backend "s3" {
+    acl            = "private"
+    bucket         = "global-tf-state-aqsvzyd5u9"
+    encrypt        = true
+    key            = "aws/analytical-platform-data-production/control-panel-message-broker/terraform.tfstate"
+    region         = "eu-west-2"
+    dynamodb_table = "global-tf-state-aqsvzyd5u9-locks"
+  }
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.11.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.5.1"
+    }
+  }
+  required_version = "~> 1.5"
+}
+
+provider "aws" {
+  alias = "session"
+}
+
+provider "aws" {
+  region = "eu-west-2"
```

Comment:
Additional components for Analytical Platform/Control Panel should reside in the same region, `eu-west-1`.

Reply:
@jacobwoffenden , I don't mind to change it to `eu-west-1`,  we used `eu-west-2` for firebreak on dev account,  and also had a short discussion with @julialawrence 
https://mojdt.slack.com/archives/C05J081AD98/p1690374209722419

Reply:
I'm not in favour of splitting control panel components geographically, what happens if eu-west-2 goes does down? does cross region communication add any latency?

I think extensions to control panel should reside in the same region.

@bagg3rs - thoughts?

Reply:
It does make sense to keep it in the same region. Keep it simple unless this was to e.g. to improve redundancy.

Reply:
ok, both, let me change it to `eu-west-1`

Reply:
Resolving as this is done.